internal/rbd/nodeserver.go

```diff
@@ -188,6 +201,11 @@ func populateRbdVol(
 	// get rbd image name from the volume journal
 	// for static volumes, the image name is actually the volume ID itself
 	if isStaticVol {
+		if req.GetVolumeContext()["migration"] == "true" {
+			// if migration static volume, use imageName as volID
+			rv.RbdImageName = req.GetVolumeContext()["imageName"]
```

Comment:
this looks redundant to me.

Reply:
the rv struct should have it , but kind of double confirmaion

Reply:
But do we need this double-check here? IMO it not needed

Reply:
imo, it looks like its not populated at this stage, I could be wrong, but making sure or assuring should not be of no harm as well.

Reply:
you are doing 
```
rv.RbdImageName = req.GetVolumeContext()["imageName"]
volID = rv.RbdImageName
```
inside the if check and again after if check there is code to assign `volID` to ` rv.RbdImageName`

this can be `volID=req.GetVolumeContext()["imageName"]` You don't need to assign it the imageName and volID and again assign volID to imageName